src-java/kilda-model/src/main/java/org/openkilda/model/bitops/cookie/ServiceCookieSchema.java

```diff
@@ -0,0 +1,157 @@
+/* Copyright 2020 Telstra Open Source
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ */
+
+package org.openkilda.model.bitops.cookie;
+
+import org.openkilda.exception.InvalidCookieException;
+import org.openkilda.model.Cookie;
+import org.openkilda.model.MeterId;
+import org.openkilda.model.bitops.BitField;
+import org.openkilda.model.bitops.NumericEnumField;
+
+import org.apache.commons.lang3.ArrayUtils;
+
+import java.util.Set;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public class ServiceCookieSchema extends CookieSchema {
+    public static final ServiceCookieSchema INSTANCE = new ServiceCookieSchema();
+
+    private static final Set<CookieType> allowedTypes = Stream.of(
+            CookieType.SERVICE_OR_FLOW_SEGMENT,
+            CookieType.LLDP,
+            CookieType.MULTI_TABLE_ISL_VLAN_EGRESS_RULES,
+            CookieType.MULTI_TABLE_ISL_VXLAN_EGRESS_RULES,
+            CookieType.MULTI_TABLE_ISL_VXLAN_TRANSIT_RULES,
+            CookieType.MULTI_TABLE_INGRESS_RULES,
+            CookieType.ARP_INPUT_CUSTOMER_TYPE
+    ).collect(Collectors.toSet());
+
+    // update ALL_FIELDS if modify fields list
+    //                   used by generic cookie -> 0x9FF0_0000_0000_0000L
```

Comment:
why `9`? 
it is `1001` as binary. should it be `7FF0`?

Reply:
```
hex(8000000000000000 | 1ff0000000000000) => 0x9ff0000000000000
bin(8000000000000000 | 1ff0000000000000) => 0b1001111111110000000000000000000000000000000000000000000000000000
```